requirements/classes/neur1630/condarc

```diff
@@ -0,0 +1,15 @@
+# channel locations. These override conda defaults, i.e., conda will
+# search *only* the channels listed here, in the order given.
+# Use "defaults" to automatically include all default channels.
+# Non-url channels will be interpreted as Anaconda.org usernames
+# (this can be changed by modifying the channel_alias key; see below).
+# The default is just 'defaults'.
+channels:
+  - conda-forge
```

Comment:
@anna-murphy You might want to add the `bioconda` channel here. Neur1630 has quite a few biology-specific packages that are only available in these channels